test/inner-loop/devfile-regression-test.sh

```diff
@@ -9,9 +14,29 @@ cd stacktest-reg
 # introduced via this current PR being tested
 echo -e "\n> Clone the main branch stack repo"
 git clone https://github.com/OpenLiberty/application-stack.git
+cd application-stack
+
+echo -e "\n> Check new stack image version"
+currentStackImage=$(cat build.sh | grep "openliberty/application-stack:")
+currentStackImageVersion=$(echo ${currentStackImage##*:} | sed 's/\}\"//')
+
+echo "Current stack image version: $currentStackImageVersion"
+echo "New stack image version: $stackImageVersion"
+
+currVerArray=( ${currentStackImageVersion//./ } )  
+newVerArray=( ${stackImageVersion//./ } ) 
```

Comment:
Interesting that when i tried this. It did not have the intended behavior. It just gives me an array of size one and if i remove the space between `/ }`, it has the correct effect of removing the periods.

Reply:
Strange... this seems to work for me. I'm not sure if removing the periods is the intention vs splitting on the periods. The periods may still persist, but the number comparison will still be valid.